apps/researcher/src/app/[locale]/objects/definitions.tsx

```diff
@@ -0,0 +1,10 @@
+export enum ListView {
+  Grid = 'grid',
+  List = 'list',
+}
+
+export enum ImageVisibility {
+  Hide = 'hide',
```

Comment:
Minor: should this be 'Invisible'. Or in other words: is 'visibility' a state ('invisible') or an action (the act of hiding)?